db/migrate/20170207155507_create_topics.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+class CreateTopics < ActiveRecord::Migration
+  def change
+    create_table :topics do |t|
+      t.string :slug, index: true
```

Comment:
Galleries are going to be filtered by theme, where it will be better to have in the URL `?topic=fashion` than `topic=3`